src/Libraries/CoreNodes/DSCoreNodesImages.resx

```diff
@@ -5119,44 +5119,6 @@
         gOTfKGA1R7m+SSE8APXA8bsfLqezAAAAAElFTkSuQmCC
 </value>
   </data>
-  <data name="DSCore.IO.File.ReadText.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
For the moment I am merging this PR, so that we can proceed with testing. But @aosyatnik needs to investigate, why obsolete nodes require icons?